### PR TITLE
Chore/update docs version compatibility

### PR DESCRIPTION
**Pull request by MilesHeise**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[x] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

did not explicitly mention version compatibility

Issue Number: N/A

## What is the new behavior?

removes ambiguity by stating Angular version compatibility

## Does this PR introduce a breaking change?

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Comment by MilesHeise** [comment]
totally unrelated note: just wanted to say I really enjoy your articles! Your pieces on structural directives and on custom decorators have both been _very_ useful recently and I am always psyched to see a new one come out

File: README.md [hunk 1]
```diff
@@ -29,7 +29,8 @@ Let's take a look at all the neat things we provide:
 ✅ Provides Helpful Methods <br>
 ✅ Typed and DRY `ControlValueAccessor` <br>
 ✅ Typed `FormBuilder` <br>
-✅ Persist the form's state to local storage
+✅ Persist the form's state to local storage <br>
+✅ Compatible with Angular versions 8+
```
Comment (NetanelBasal) [inline README.md:33]: It's not the right place

Comment (MilesHeise) [reply README.md:33]: happy to update, where do you think this information would best fit?

Comment (MilesHeise) [reply README.md:33]: perhaps a 'compatibility' section at the end right after the 'migration' section? that was the other one of my two first thoughts where to put it

Comment (NetanelBasal) [reply README.md:33]: Sounds good

Comment (NetanelBasal) [reply README.md:33]: Also, add yourself as a contributor.

Comment (MilesHeise) [reply README.md]: updated, cheers!

**Pull request merged**